File: Portal/Models/GameModel.cs
using System.Collections.Generic;

namespace Descent.Web.Models
{
    public class GameModel
    {
        public string Id { get; set; }
        public string User { get; set; }
        public string Name { get; set; }
        public string Notes { get; set; }
        public string CharacterImage { get; set; }
        public string Gold { get; set; }
        public string Class { get; set; }
        public string ClassType { get; set; }
        public List<UserModel> Users { get; set; }
        public List<CardModel> Objects { get; set; }
        public List<CardModel> Skills { get; set; }
    }
}

File: Portal/tsScripts/shared/model/game.model.ts
import { UserDefinition } from './user.model';
import { CardDefinition } from './card.model';

export class GameModelDefinition {
    Id: string;
    User: string;
    Users: Array<UserDefinition>;
    Name: string;
    CharacterImage: string;
    Gold: string;
    Objects: Array<CardDefinition>;
    Skills: Array<CardDefinition>;
    Notes: string;
}